fileutils/filetype.go

```diff
@@ -1,9 +1,11 @@
-package llm
+package fileutils
```

Comment:
I'd probably leave this as `llm` for this PR to keep the changes smaller, `llm` has a bunch of moving parts in it that might be easier to change in a follow up.